file/tests/test.bats

```diff
@@ -0,0 +1,6 @@
+source "${BATS_TEST_DIRNAME}/../plan.sh"
+
+@test "Version matches" {
+  result="$(file --version | head -1 | tr -d 'file-')"
```

Comment:
this line right here - calling of`file` is clashing with `file` that exists in the studio.  We could specify you want `/bin/file` or actual `$(hab pkg path ${HAB_ORIGIN}/file)/bin/file`

fyi:

```
# file --version
file-5.34
magic file from /hab/pkgs/core/file/5.34/20190115003731/share/misc/magic
```